app/src/main/java/com/example/oddhours/ui/home/HomeAdapter.kt

```diff
@@ -113,6 +116,9 @@ class HomeAdapter(private var jobList: List<JobModel>, val context: Context, pri
                     // DEBUG TODO: remove this eventually
                     // Log.d(TAG, "startDate: year: $year, month: ${monthOfYear}, day: $dayOfMonth")
                     // Log.d(TAG, "$startDate")
+                    dayOfYear = 0
+                    calculateDayOfTheYear(monthOfYear, dayOfMonth)
```

Comment:
maybe this can use the same method from the helper class? so we don't duplicate code / effort here too

Reply:
yaa for sure, will modify that.